onboarding/README.md

```diff
@@ -0,0 +1,60 @@
+# New Icalier Onboarding
+
+This document will guide you through the process of giving a warm welcome to new members, or at least give you a nice starting point:
+
+The onboarding process takes up to 1 week, here is how the week would look like:
+
+#### Monday
+
+This day is all about culture, the new members will spend the whole day with the cultural team. The purpose of this day is to explain some of our rituals, work dynamic and get to know other members. 
+
+#### Tuesday
+
+On this day the operations manager will step in and give an introduction on how Icalia works from a operations-like perspective, some topics to cover are:
+
+* What is the process a client lives with Icalia Labs?
+  * Discovery Session
+  * Design Sprint
+  * Minimum Viable Product Development
+  * Release
+* How the design & development team work together?
+* What are the tools we use on a daily basis?
+  * Slack
+  * Github
+  * Pivotal Tracker
+  * Email & Calendar
+* How to deal with a client?
+  * Schedule meetings
+  * Schedule demo days
+  * Schedule weekly standups
+
```

Comment:
@kurenn we should add this:

If there is time left, show them the Icalia Guides on github and tell them to read for first time.

Reply:
Playbook included.

Reply:
Once it is released we can come back and visit this, how about that?